src/Mobs/Ocelot.cpp

```diff
@@ -218,3 +218,8 @@ bool cOcelot::IsCatSittingOnBlock(cWorld * a_World, Vector3d a_BlockPosition)
 
 
 
+
+void cOcelot::HandleFalling(void)
+{
+	// empty - Ocelots don't take fall damage
```

Comment:
This function did a lot more than just do damage.  A better way to do damage immunity this is to override `DoTakeDamage` and not call `super::DoTakeDamage` for `dtFalling`.

Reply:
A similar method was used in #4136, so you may want to look there for any guidance.

Reply:
Should I change the fall damage logic for chickens too? (It's done the same way as this PR)

Reply:
I think that would be best.

Reply:
Done